Shopify/ruby-lsp
# Add Show Output Channel command for VSCode extension

### Motivation

Closes https://github.com/Shopify/ruby-lsp/issues/2958

### Implementation
- Added a new command to package.json for `showOutput`
- Utilized the existing client in the workspace to open up the output channel

### Automated Tests
No automated tests - I did not see any instances of automated testing for similar commands (`collectRubyLspInfo` for example). I'd be happy to add some but I didn't really see a pattern for something as straight forward as this.

### Manual Tests
- Open VS Code -> Run and Debug -> Run Extension
- Ensure existing Ruby LSP is disabled in new VS Code window
- Open the command palette and search for `Ruby LSP: Show Output Channel`

https://github.com/user-attachments/assets/e777b1ac-684e-47cc-874b-cad8e5049ae4
